pkg/sparrow/targets/manager.go

```diff
@@ -230,7 +230,9 @@ func (t *manager) refreshTargets(ctx context.Context) error {
 
 	// filter unhealthy targets - this may be removed in the future
 	for _, target := range targets {
-		if !t.registered && target.Url == fmt.Sprintf("https://%s", t.name) {
+		// FIXME: this should be a more reliable check ideally
+		// this would report false negatives if sparrows dns stays the same but scheme change from http to https
```

Comment:
@y-eight and I talked about it this morning. Sparrow should overwrite the file when registering anyways, since the filename ignores the scheme and only consists of the dnsName